### PR TITLE
New version of createTissueSpecific

### PR DESCRIPTION
**Pull request by arichelle**
New version without default processing of data and update of some of the method calls to be in agreement with this new version

Note : I think it could be interesting to propose some script to facilitate the preprocessing by the user (e.g. parser for GPR rules, etc...), will add that in a new PR asap.

### CODE REVIEW

File: src/dataIntegration/transcriptomics/FASTCORE/fastcore.m [hunk 1]
```diff
@@ -1,5 +1,5 @@
 function tissueModel = fastcore(model, core, epsilon, printlevel)
-% Use the FASTCORE algorithm ('Vlassis et al, 2014') to extract a context
+% Use the FASTCORE algorithm (Vlassis et al, 2014*) to extract a context
```
Comment (laurentheirendt) [inline src/dataIntegration/transcriptomics/FASTCORE/fastcore.m]: It seems like you are having an old version here, no?

File: src/dataIntegration/transcriptomics/FASTCORE/fastcore.m [hunk 2]
```diff
@@ -45,7 +45,7 @@
     model_orig = model;
 
     %Find irreversible reactions
-    irrevRxns = find(model.rev>=0);
+    irrevRxns = find(model.rev==0);
```
Comment (laurentheirendt) [inline src/dataIntegration/transcriptomics/FASTCORE/fastcore.m]: I guess that this is an old version ...?

Comment (arichelle) [inline src/dataIntegration/transcriptomics/FASTCORE/fastcore.m]: Arf, I always forget to keep that :/ really sorry

Comment (laurentheirendt) [inline src/dataIntegration/transcriptomics/FASTCORE/fastcore.m]: May you also keep the header of the function that you already formatted? Thanks 😄

**Comment by laurentheirendt** [comment]
@arichelle, the original `INIT.m` version has been moved to the `src` folder as discussed.

Is the version of `INIT.m` in this PR a new one?

**Comment by arichelle** [comment]
The version of INIT is the same but i've modified some options to fit with the new version of createTissueSpecific

**Comment by tpfau** [comment]
Hi  @arichelle, thanks a lot for this. :tada: 
The only thing that will need some change is the documentation (as we don't currently have additional parameters being parsed automatically). 
The easiest (for now) would be to just change it to:
"NOTE: "
because thats a known field for the documentation parser.

File: src/dataIntegration/transcriptomics/FASTCORE/fastcore.m (hunk 2 above)
Comment (tpfau) [inline src/dataIntegration/transcriptomics/FASTCORE/fastcore.m]: Btw: this shouldn't use rev but lb >= 0

**Comment by arichelle** [comment]
I think I've performed all the requested changes, let me know if I need to perform any other modification

**Comment by tpfau** [comment]
Should be fine I think.

**Comment by laurentheirendt** [comment]
Thanks @arichelle and @tpfau 🌵

**Pull request merged**